lab-kyler/test/greet-test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const expect = require('chai').expect
+const kgreet = require('../greet.js');
+const assert = require('assert');
+
+describe('Greet module', function() {
+  describe('#greet', function() {
+    it('should return "Hello, Newman!"', function () {
+      //assert.ok(kgreet.greet('Newman') === 'Hello, Newman!', 'Not equal to "Hello, Newman!"');
+      expect(kgreet.greet('Newman')).equal('Hello, Newman!');
+    });
+
+    it('should throw an error', function () {
+      //assert.throws(function() {
+      //  kgreet.greet();
+      //}, 'No error thrown!"');
+      expect(function() {
+         kgreet.greet()
+      }).to.throw(Error);
+    });
+
+    it('1+1=2 test', function() {
+        expect(1+1).equal(2);
+    });
+  });
+});
```

Comment:
Instead of kgreet you could have used greetings or some other variable name that's more clear in its meaning, that's just a nitpick though. I'm glad to see you making use of chai for your tests. I notice you didn't use any chain getters which is fine, however, I think we will be using them in class for more readable, verbose tests :)